src/lines-component.coffee

```diff
@@ -343,59 +114,18 @@ class LinesComponent
 
   measureCharactersInNewLines: ->
     @presenter.batchCharacterMeasurement =>
-      for id, lineState of @oldState.lines
-        unless @measuredLines.has(id)
-          lineNode = @lineNodesByLineId[id]
-          @measureCharactersInLine(id, lineState, lineNode)
-      return
```

Comment:
Should this `return` be retained so that the results of the `for` loop aren't built up in an array and returned?